packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -93,29 +116,36 @@ mod tests {
     use super::*;
     use crate::types::EntityType;
 
-    const CONNECTION_STRING: &str = "postgres://postgres:postgres@localhost/postgres_graph";
+    const USER: &str = "postgres";
+    const PASSWORD: &str = "postgres";
+    const HOST: &str = "localhost";
+    const PORT: u16 = 5432;
+    const DATABASE: &str = "graph";
 
     // TODO - long term we likely want to gate these behind config or something, probably do not
     //  want to add a dependency on the external service for *unit* tests
 
     #[ignore]
     #[tokio::test]
-    async fn can_connect() {
-        PostgresDatabase::new(CONNECTION_STRING)
-            .await
-            .expect("Couldn't connect to the Postgres DB");
+    async fn can_connect() -> Result<(), sqlx::Error> {
+        PostgresDatabase::new(USER, PASSWORD, HOST, PORT, DATABASE).await?;
```

Comment:
Just curious, was the removal of `.expect(...)` intentional here?

Reply:
Yes. I changed the function signature to return a `Result`, so it's possible to use the `?` operator instead of `.expect()`